alicloud/provider.go

```diff
@@ -77,6 +80,14 @@ func Provider() terraform.ResourceProvider {
 				Deprecated: "Field 'fc' has been deprecated from provider version 1.28.0. New field 'fc' which in nested endpoints instead.",
 			},
 			"endpoints": endpointsSchema(),
+			"profile_path": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"profile_name": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
```

Comment:
Please add description for there parameters and you can refer to https://github.com/terraform-providers/terraform-provider-aws/blob/master/aws/provider.go#L38 and https://github.com/terraform-providers/terraform-provider-aws/blob/master/aws/provider.go#L47

In addition, please rename `profile_path` to shared_credentials_file and `profile_name` to profile.